toolbar-button/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@jupyterlab-examples/toolbar-button",
+  "version": "0.1.0",
+  "description": "A JupyterLab extension for clearing all cells output at once.",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/jupyterlab/extension-examples",
+  "bugs": {
+    "url": "https://github.com/jupyterlab/extension-examples/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Yash Singhal",
+    "email": "ramusinghal112@gmail.com"
+  },
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,.js,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jupyterlab/extension-examples.git"
+  },
+  "scripts": {
+    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build:prod": "jlpm run clean && jlpm run build:lib && jlpm run build:labextension",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf clear_cell_outputs/labextension",
```

I think this folder needs to match the one declared in the `outputDir` below:

```suggestion
    "clean:labextension": "rimraf jupyterlab_examples_toolbar_button/labextension",
```